cmd/dumpling/main.go

```diff
@@ -77,6 +79,7 @@ func init() {
 	rootCmd.PersistentFlags().StringVar(&statusAddr, "status-addr", ":8281", "dumpling API server and pprof addr")
 	rootCmd.PersistentFlags().Uint64VarP(&rows, "rows", "r", export.UnspecifiedSize, "Split table into chunks of this many rows, default unlimited")
 	rootCmd.PersistentFlags().StringVar(&where, "where", "", "Dump only selected records")
+	rootCmd.PersistentFlags().BoolVar(&escapeBackslash, "escape-backslash", false, "use backslash to escape quotation marks")
```

Comment:
The default should be `true` and has the same meaning of (inverse of) [`NO_BACKSLASH_ESCAPES`](https://dev.mysql.com/doc/refman/8.0/en/sql-mode.html#sqlmode_no_backslash_escapes).

If `escape-backslash=true`, replace `'` by `\'` and `\` by `\\` (and handle `\0`, `\n`, etc. if needed)

If `escape-backslash=false`, replace `'` by `''` and don't touch anything else.

Reply:
Revised, PTAL